catalog/plugin/catalog-plugin-oauth/src/main/java/org/codice/ddf/catalog/plugin/oauth/OAuthPlugin.java

```diff
@@ -140,17 +141,22 @@ public QueryRequest process(Source source, QueryRequest input) throws StopProces
 
     Object securityAssertion = input.getProperties().get(SECURITY_SUBJECT);
     if (!(securityAssertion instanceof Subject)) {
-      LOGGER.warn("A user Subject is not available.");
-      throw new StopProcessingException("A user Subject is not available.");
+      LOGGER.warn("The user's subject is not available.");
+      throw new StopProcessingException("The user's subject is not available.");
     }
 
     Subject subject = (Subject) securityAssertion;
-    String userId = SubjectUtils.getEmailAddress(subject);
-    if (userId == null) {
-      userId = SubjectUtils.getName(subject);
+    Session session = subject.getSession();
+    if (session == null) {
+      LOGGER.warn("The user's session is not available.");
+      throw new StopProcessingException("The user's session is not available.");
     }
 
-    TokenInformation.TokenEntry tokenEntry = tokenStorage.read(userId, oauthSource.getId());
+    String sessionId = session.getHost();
```

Comment:
https://github.com/codice/ddf/blob/bac03b6b342e4ae3db348dddfa5fa30b49528cda/platform/security/core/security-core-services/src/main/java/ddf/security/service/impl/SecurityManagerImpl.java#L65
☝️ sets it on the host.

Reply:
That's so confusing. There's a field on the class for the id even! I'm not gonna block since it was already like that, but imo we should use the SimpleSession::setId instead of the constructor so it's more clear.

Reply:
We set it, I'll change it 👍